.github/workflows/docker-build-ethermint-debug.yml

```diff
@@ -40,10 +40,10 @@ jobs:
           ls
           mv ethermint ..
           cd ..
-          cp -R optimint ethermint
+          cp -R rollmint ethermint
```

Comment:
nit: we are renaming it once which means we might rename it again. Or if someone forks this and wants to name it differently they are going to need to find and replace again. 

So could we just use a `env` variable up top? Like `DIR_NAME`?

You could then use that var in a number of places.

Reply:
Created #546,